core/src/test/java/io/questdb/test/griffin/wal/AbstractWalWriterFuzzTest.java

```diff
@@ -0,0 +1,491 @@
+package io.questdb.test.griffin.wal;
+
+import io.questdb.cairo.*;
+import io.questdb.cairo.wal.ApplyWal2TableJob;
+import io.questdb.cairo.wal.CheckWalTransactionsJob;
+import io.questdb.cairo.wal.WalPurgeJob;
+import io.questdb.cairo.wal.WalWriter;
+import io.questdb.cairo.wal.seq.TableSequencerAPI;
+import io.questdb.griffin.SqlException;
+import io.questdb.griffin.model.IntervalUtils;
+import io.questdb.mp.WorkerPool;
+import io.questdb.std.*;
+import io.questdb.std.datetime.microtime.Timestamps;
+import io.questdb.std.str.Path;
+import io.questdb.test.AbstractGriffinTest;
+import io.questdb.test.fuzz.FuzzTransaction;
+import io.questdb.test.fuzz.FuzzTransactionOperation;
+import io.questdb.test.mp.TestWorkerPool;
+import io.questdb.test.tools.TestUtils;
+import org.jetbrains.annotations.NotNull;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+public abstract class AbstractWalWriterFuzzTest extends AbstractFuzzTest {
```

Comment:
Given the amount of WAL specific logic in there, it seemed like a better idea to avoid having `AbstractFuzzTest` be even larger.

I can flatten everything up, but then there'll be extra unnecessary code for the `FrameAppendFuzzTest` which also derives from `AbstractFuzzTest`.

@ideoma, can you confirm you prefer me to flatten it?

Reply:
Discussed in person. Will flatten.